c++/src/capnp/c++.capnp

```diff
@@ -24,3 +24,13 @@ $namespace("capnp::annotations");
 
 annotation namespace(file): Text;
 annotation name(field, enumerant, struct, enum, interface, method, param, group, union): Text;
+
+annotation omitSchemas(file): Void;
+# Requests that the generated code does not embed the types' schemas.
+# This means you will not be able to use <capnp/schema.h> or <capnp/dynamic.h> APIs with this type.
+# By extension, that means you will not be able to stringify messages nor use the JSON codec.
+# However, the generated code will be somewhat smaller and will not include easily-extractable
+# type information, which may be important in some use cases.
+#
+# All files that import your file will also be required to set this option if they are used to
+# generate C++ code.
```

Comment:
Missing newline at end of file.

(This may seem nitpicky, but it's problematic mostly because so many tools complain about it, like the review tools here. The reason they all complain is because some old Unix standard actually says that text files must end with a newline, though obviously nothing enforces that in reality.)

Annoyingly it doesn't look like GitHub's suggested changes interface lets me suggest adding the newline... it complains that I haven't changed anything, unless I had two newlines...